stack.yaml

```diff
@@ -49,6 +49,10 @@ extra-deps:
 - persistent-sqlite-2.11.1.0
 - persistent-template-2.9.1.0@rev:2
 - esqueleto-3.4.1.1
+- duckling-0.2.0.0
+- dependent-sum-0.7.1.0
+- constraints-extras-0.3.1.0
+- string-0.4.0
```

Comment:
you don't need `string` here (although you may need to `stack clean` to persuade it)